extensions/amp-pinterest/0.1/test/test-amp-pinterest.js

```diff
@@ -44,8 +51,8 @@ describes.realWin(
       pin.setAttribute('data-media', pinMedia);
       pin.setAttribute('data-description', pinDescription);
       pin.setAttribute('layout', 'responsive');
-      pin.setAttribute('width', '100');
-      pin.setAttribute('height', '100');
+      pin.setAttribute('width', width);
```

Comment:
This seems to be unrelated to the `aria-label`, do you want to include this in the PR?

If so, we should likely add some tests for the default values.